proxy/api/src/main.rs

```diff
@@ -83,15 +83,51 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let bin_dir = config::bin_dir()?;
     coco::git_helper::setup(&proxy_path, &bin_dir).expect("Git remote helper setup failed");
 
-    log::info!("Starting API");
     let ctx = context::Ctx::from(context::Context {
         peer_api,
         keystore,
         store,
     });
+
+    {
+        let ctx = ctx.clone();
+        log::info!("Starting Announcement watcher");
+        tokio::task::spawn(async move { announcement_watcher(ctx).await });
```

Comment:
I'm probably being a noob, but I would have thought that this was equivalent to:
```
tokio::task::spawn(announcement_watcher(ctx));
```

Reply:
Nope, I am. Just forgot to remove the closure during refactoring.

Reply:
Ah cool, I think there's another one of those somewhere if I'm not mistaken :thought_balloon: